ament_mypy/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ament_mypy</name>
+  <version>0.7.3</version>
+  <description>Support for mypy static type checking in ament.</description>
+  <maintainer email="ted.kern@canonical.com">Ted Kern</maintainer>
+  <license>Apache License 2.0</license>
+  <author email="ted.kern@canonical.com">Ted Kern</author>
+
+  <exec-depend>python3-mypy</exec-depend>
```

Comment:
This tag is invalid and fails to validate against the referenced schema. The tag name should be `exec_depend`.

Reply:
Handled in d4130a2